### PR TITLE
ft(landing page setup): implement design of landing page

### PR DESCRIPTION
**Pull request by Ebunwadi**

### CODE REVIEW

File: src/pages/Landing-page/LandingPage.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import Navbar from '../../components/landing-page-components/Navbar'
+import Main from '../../components/landing-page-components/Main'
+
+function LandingPage() {
+  return (
+    <div className='landing-page'>
+      <Navbar />
```
Comment (courage173) [inline src/pages/Landing-page/LandingPage.js:8]: This NavBar is going to be in all the routes. adding it just on the landing page might not make sense to add it here. have an higher Order function to wrap them.

File: src/components/landing-page-components/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+// import '../../pages/landing-page/navbar.css'
```
Comment (courage173) [inline src/components/landing-page-components/Navbar.jsx]: comments

**Review by courage173 — CHANGES_REQUESTED** [review]
theses are some changes you need to make.
1. Change to use css modules. you can research on how to do that
2. Ensure you deploy on netlify so we can also have a review url for each PR (pull request)
3. have an assets folder where you put images
4. Create an higher order component called a layout where you put in the header and can wrap other components.

File: src/pages/Landing-page/LandingPage.js (hunk 1 above)
Comment (Ebunwadi) [reply src/pages/Landing-page/LandingPage.js:8]: okay boss

**Review by courage173 — APPROVED** [review]
You can merge now

**Comment by courage173** [comment]
You have not added the fonts link

**Pull request merged**